internal/tui/bubble.go

```diff
@@ -7,12 +7,18 @@ import (
 	tea "github.com/charmbracelet/bubbletea"
 	"github.com/charmbracelet/lipgloss"
 	"github.com/charmbracelet/soft-serve/internal/config"
+	"github.com/charmbracelet/soft-serve/internal/tui/bubbles/git/types"
+	gitypes "github.com/charmbracelet/soft-serve/internal/tui/bubbles/git/types"
```

Comment:
imported 2 times...

I would also alias to `gittypes` instead of `gitypes`

Reply:
done